parser/class_files.py

```diff
@@ -242,4 +243,33 @@ def parse_remarks(tag, target, dita_soup, options):
         print(f"{options['file_path']} does not have a div element with h1 named REMARKS")
 
 
+def parse_non_class_file(tag, file_path):
+    #Check if there is an html <div id="QuickLinksTable"> </div>
```

Comment:
Hello @AbegaM - this next block of text should be outside this function.

We have assorted logic to find file to handle (propulsion, remarks, others), either from QuickLinksTable or just an `<a>` in html content.

This method should just receive a file - and it will then convert it.

The external logic will then decide what to do with the href for the converted file.  It _may_ put it into `PropulsionRef` or it _may_ put it into `related-pages`, or it _may_ just include it as the `href` for content in the middle of a paragraph of html.